app/javascript/packages/eslint-plugin/lib/is-installed.js

```diff
@@ -3,7 +3,9 @@ const { readFileSync } = require('fs');
 
 let cache;
 
-function isInstalled(name) {
+function isInstalled(nameOrNames) {
```

Comment:
WDYT of splitting the array version into a separate function `areAllInstalled`? Looking at the use cases above I couldn't tell if it was an "any" or any "every" without digging into source

Reply:
Yeah, in fact, maybe I'll just make the array form the _only_ form.

Reply:
Tried it in 187d86939, didn't much care for it stylistically, so went the opposite direction in f8c100542 and reverted to singular form, more like what exists in `main`.

Reply:
makes sense! super clear now